symengine/sets.h

```diff
@@ -249,6 +249,39 @@ class Complement : public Set
     }
 };
 
+class ConditionSet : public Set
+{
+private:
+    vec_sym syms_;
```

Comment:
Yes, you can make it a single variable and you can use a FiniteSet (which is a single entity) if there is a need